docs/DML-Operations-on-Carbon.md

```diff
@@ -91,12 +91,17 @@ Following are the options that can be used in load data:
     ```ruby
     OPTIONS('ALL_DICTIONARY_PATH'='/opt/alldictionary/data.dictionary')
     ```
-- **COLUMNDICT:** dictionary file path for single column.
+- **COLUMNDICT:** Dictionary file path for each column.
 
     ```ruby
     OPTIONS('COLUMNDICT'='column1:dictionaryFilePath1, column2:dictionaryFilePath2')
     ```
     Note: ALL_DICTIONARY_PATH and COLUMNDICT can't be used together.
+- **DATEFORMAT:** Date format for each column.
+
+    ```ruby
+    OPTIONS('DATEFORMAT'='column1:dateFormat1, column2:dateFormat2')
```

Comment:
I add a note, ref to the JAVA SimpleDateFormat Class Doc. It provides more details.